columnphysics/icepack_atmo.F90

```diff
@@ -359,8 +359,16 @@ subroutine atmo_boundary_layer (sfctype,            &
       ! as in Jordan et al (JGR, 1999)
       !------------------------------------------------------------
 
-      shcoef = rhoa * ustar * cp * rh + c1
-      lhcoef = rhoa * ustar * Lheat  * re
+      if (heatflux_linear) then
+         !- Use constant coefficients for sensible and latent heat fluxes
+         !    similar to atmo_boundary_const but using (wind-Vice) instead of wind-only
+         shcoef = (1.20e-3_dbl_kind)*cp_air*rhoa*vmag
+         lhcoef = (1.50e-3_dbl_kind)*Lheat *rhoa*vmag
+      else
+         !- Use Jordan et al (JGR, 1999) formulation
```

Comment:
Please remove the ref to Jordan in the comment.  That's referring to a small modification of the overall turbulent heat flux calculation, which came from somewhere else (documented in readthedocs, I hope).

Reply:
Done...